atomicapp/constants.py

```diff
@@ -37,6 +37,7 @@
 ANSWERS_FILE_SAMPLE_FORMAT = 'ini'
 WORKDIR = ".workdir"
 LOCK_FILE = "/run/lock/atomicapp.lock"
+LOG_FILE = "/var/log/atomicapp.log"
```

Comment:
I'd suggest using `Utils.getRoot()` https://github.com/projectatomic/atomicapp/blob/master/atomicapp/utils.py#L213 and log to host if it's available.

And maybe condition the file logging with some argument (`--persistent-log`, `--log-file`...?)